src/Cli.Tests/ValidateConfigTests.cs

```diff
@@ -114,6 +114,37 @@ public void TestValidateWithEmptyConfig()
         }
     }
 
+    /// <summary>
+    /// This Test is used to verify that the validate command is able to catch invalid values for the depth-limit property.
+    /// </summary>
+    [DataTestMethod]
+    [DataRow("null", true, DisplayName = "Invalid Value: 'null'. Only integer values are allowed.")]
```

Comment:
is there a test where you have null without quotes?
```json
{
    "max-depth": null
}
```

Reply:
added a test for that.